.github/workflows/test_pull_request.yml

```diff
@@ -0,0 +1,18 @@
+name: Swift
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Build and Run tests
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: swift build -v
```

Comment:
not sure whether this is required. When executing test -v, it also needs to build the project.